### DISCUSSION TITLE
Switch State after reboot "true"

### DISCUSSION THREAD

**Discussion by Shadesh** [arendst/Tasmota#15215] [Q&A] [answered]
Hi, 

I got a strange problem with Tasmota32. I configured 3 switches, two of them basically just a magnetic switch for my garage. One for open, one for closed. The thing is, after reboot all of them are in state "true". As soon as I let the door open and close it triggers the switch3 and it goes to state "false". I found https://github.com/arendst/Tasmota/pull/3910, and I used the rule there as:

``rule1 on switch2#boot do event bootsw2=%value% endon on switch3#boot do event bootsw3=%value% endon on system#boot do event bootsys endon``

it seems the rule works but the value is "1" for both, as I can see in the webconsole during boot, but switch3 shouldn't be "true"  because there is no "closed circuit". My switch mode is for both "1" and I also used SetOption114 because I use a relay on the same module.

Any ideas what im doing wrong here? Thanks a lot!

**Comment by sfromis** [comment, marked as answer]
Do not assume some logical relationship between terms on/off, true/false, high/low, open/closed, connected/not, pushed/not and so on, except that a Tasmota state of "on" generally means input signal high (3.3V / pullup). For a "normal" input, like a simple Button pin, input high means not active, while input low (gnd) means activated.

**Reply by Shadesh** [reply]
I never found out why this problem existed. Did a fresh erased install and configured exactly the same, now switch3 stays false even after a reboot.
